api/app/tests/jobs/job_fixtures.py

```diff
@@ -52,6 +52,8 @@ def mock_wfwx_stations():
     station_2 = WFWXWeatherStation(latitude=1, longitude=1, elevation=1,
                                    wfwx_id='ba28973a-0a79-04ea-e053-1d09228e8c65', code=2,
                                    name='blah', zone_code='T1')
+    loop = asyncio.new_event_loop()
```

Comment:
The test that exercises this started complaining there was no event loop (only on CI!) when constructing the `Future` below, so it's explicitly created now.